monitoring/local_monitoring/local_monitoring_demo.sh

```diff
@@ -0,0 +1,289 @@
+	echo "Info: Access kruize UI at http://${KRUIZE_UI_URL}"
+	echo "Info: List all Kruize Experiments at http://${KRUIZE_URL}/listExperiments"
+	echo
+}
+
+#
+# "local" flag is turned off by default for now. This needs to be set to true.
+#
+function kruize_local_patch() {
+	CRC_DIR="./manifests/crc/default-db-included-installation"
+	KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT="${CRC_DIR}/openshift/kruize-crc-openshift.yaml"
+	KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE="${CRC_DIR}/minikube/kruize-crc-minikube.yaml"
+
+	pushd autotune >/dev/null
+		# Checkout mvp_demo to get the latest mvp_demo release version
+		git checkout mvp_demo >/dev/null 2>/dev/null
+
+		if [ ${CLUSTER_TYPE} == "minikube" ]; then
+			sed -i 's/"local": "false"/"local": "true"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE}
+		elif [ ${CLUSTER_TYPE} == "openshift" ]; then
+			sed -i 's/"local": "false"/"local": "true"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT}
+		fi
+	popd >/dev/null
+}
+
+#
+#
+#
+function kruize_local_demo_setup() {
+	# Start all the installs
+	start_time=$(get_date)
+	echo
+	echo "#######################################"
+	echo "#       Kruize Local Demo Setup       #"
+	echo "#######################################"
+	echo
+
+	if [ ${kruize_restart} -eq 0 ]; then
+		clone_repos autotune
+		clone_repos benchmarks
+		if [ ${CLUSTER_TYPE} == "minikube" ]; then
+			minikube >/dev/null
+			check_err "ERROR: minikube not installed"
+			minikube_start
+			prometheus_install autotune
+		fi
+		benchmarks_install
+	fi
+	kruize_local_patch
+	kruize_install
+	echo
+
+	kubectl_cmd="kubectl -n default"
+	TECHEMPOWER_PORT=$(${kubectl_cmd} get svc tfb-qrh-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+	TECHEMPOWER_IP=$(${kubectl_cmd} get pods -l=app=tfb-qrh-deployment -o wide -o=custom-columns=NODE:.spec.nodeName --no-headers)
+
+	if [ ${CLUSTER_TYPE} == "minikube" ]; then
+		kubectl_cmd="kubectl -n monitoring"
+
+		MINIKUBE_IP=$(minikube ip)
+
+		KRUIZE_PORT=$(${kubectl_cmd} get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+		KRUIZE_UI_PORT=$(${kubectl_cmd} get svc kruize-ui-nginx-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+
+		export KRUIZE_URL="${MINIKUBE_IP}:${KRUIZE_PORT}"
+		export KRUIZE_UI_URL="${MINIKUBE_IP}:${KRUIZE_UI_PORT}"
+		export TECHEMPOWER_URL="${MINIKUBE_IP}:${TECHEMPOWER_PORT}"
+	elif [ ${CLUSTER_TYPE} == "openshift" ]; then
+		kubectl_cmd="kubectl -n openshift-tuning"
+
+		KRUIZE_IP=$(${kubectl_cmd} get pods -l=app=kruize -o wide -o=custom-columns=NODE:.spec.nodeName --no-headers)
+		KRUIZE_UI_IP=$(${kubectl_cmd} get pods -l=app=kruize-ui-nginx -o wide -o=custom-columns=NODE:.spec.nodeName --no-headers)
+
+		KRUIZE_PORT=$(${kubectl_cmd} get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+		KRUIZE_UI_PORT=$(${kubectl_cmd} get svc kruize-ui-nginx-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+
+		export KRUIZE_URL="${KRUIZE_IP}:${KRUIZE_PORT}"
```

Comment:
IP and PORT didn't work on AWS cluster, should we expose the kruize service and use the route? I think this will work on all OCP clusters